home_page/implications/script.js

```diff
@@ -333,23 +333,46 @@ function renderImplications(index) {
     const eq = Equation.fromId(eqId);
     updateUrl(eqId);
 
+    let equationHasCommentary = false;
```

Comment:
I don't like introducing these new variables that make it more difficult to track what's going on and what means what. Just checking `if (commentary[bigIndex])` is easier to understand.